tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,10 @@
 const App = () => {
-	return <div>A tic-tac-toe game</div>;
+	return (
+		<div>
+			<h1 className='header'>Tic-Tac-Toe</h1>
+			<Game />
```

Comment:
The components in this code base each have their own module. 👍 
This way of organizing code allows for other developers to easily access your code and more efficiently understand the purpose of each component.